app/views/landing/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
```

Comment:
The layout file is in haml, but this is in erb. Write this one in haml, stay consistent.